lib/Minion/Backend/Pg.pm

```diff
@@ -721,6 +721,12 @@ These fields are currently available:
 
 =over 2
 
+=item id
+
+  id => 1
+
+Lock id.
+
```

Comment:
There is multiple places where this needs to be added. Also the fields are in alphabetical order.

Reply:
Updated Backend.pm, I think this was the only other place I neede